pkgs/development/interpreters/mujs/default.nix

```diff
@@ -9,6 +9,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-ZpdtHgajUnVKI0Kvc9Guy7U8x82uK2jNoBO33c+SMjM=";
   };
 
+  patches = [
```

Comment:
How do you feel about also backporting https://git.ghostscript.com/?p=mujs.git;a=commit;h=160ae29578054dc09fd91e5401ef040d52797e61 ?

There is no CVE ID attached to it but the commit message is not really reassuring?

Reply:
Yup, patch looks quite sensible & self contained and it applies cleanly.